cfep-03.md

```diff
@@ -0,0 +1,72 @@
+
+<table>
+<tr><td> Title </td><td> Providing manually uploaded builds on the conda-forge anaconda channel</td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Hadrien Mary &lt;hadrien.mary@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Sept 27, 2016</td></tr>
+<tr><td> Updated </td><td> Sept 27, 2016</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+Some packages are very hard to build on the currently used CI (CircleCI, Travis and AppVeyor). It's most of the time because of time limitations in the CI website and also because of the complexity of the build.
```

Comment:
On this maybe @ccordoba12 could describe more precisely why does it take so long to get the Qt build.

Naively I would say that the huge amount of C/C++ dependencies in a project make the process to build it harder than without dependencies. And if you add the fact that you have to wait for the CI to end before being able to do a modification and try to compile again, it makes the whole process discouraging.